wikidataQuery.py

```diff
@@ -24,7 +25,10 @@ def searchEntity(entity, string_type):
     json = requests.get(url,params).json()
     
     # Return the most likely entity
-    return json['search'][0]['id']
+    if len(json['search']) > 0:
+        if settings.verbose:
+            print(json['search'][0]['id'])
+        return json['search'][0]['id']
```

Comment:
Note: I realise this line is actually out-of-scope of this PR. However, since it needed to be done anyway, I thought I might as well include it while working on it anyway.